Resources/config/core.xml

```diff
@@ -28,6 +28,7 @@
             <argument>%sonata.admin.configuration.breadcrumbs%</argument>
         </service>
         <service id="sonata.admin.default_route_id_handler" class="Sonata\AdminBundle\Route\RouteIdHandler"/>
+        <service id="sonata.admin.route_id_handler" alias="sonata.admin.default_route_id_handler"/>
```

Comment:
Also, you must provide a way to change the alias to another service through configuration.